### PR TITLE
Added meme feature and some changes

### PR DESCRIPTION
**Pull request by shasherazi**
- Added guild_id and client_id as environmental variables
- Fixed meme.add bugs

### CODE REVIEW

File: src/modules/ping.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import { SlashCommandBuilder } from '@discordjs/builders';
 import { registerCommandModule } from '../utils/commands.js';
 
-const pingCommand = new SlashCommandBuilder()
-  .setDescription('replies with pong');
+const pingCommand = new SlashCommandBuilder().setDescription(
```
Comment (copleykj) [inline src/modules/ping.js:4]: Try not to include unnecessary formatting changes in your commits.

File: src/utils/commands.js [hunk 2]
```diff
@@ -15,8 +15,8 @@ export const registerCommandModule = (name, command, handler) => {
 
 export const registerGuildCommands = async () => {
   // TODO: Replace these with ENV variables
-  const clientId = '873205734228234290';
-  const guildId = '864894667774361623';
+  const clientId = process.env.CLIENT_ID;
```
Comment (copleykj) [inline src/utils/commands.js:18]: This is great, but in the future you should try to keep unrelated changes relegated to separate commits.

**Review by copleykj — APPROVED** [review]
For the most part this looks great. I'm going to merge this. Read comments for future consideration.

File: src/modules/ping.js (hunk 1 above)
Comment (shasherazi) [reply src/modules/ping.js:4]: Didn't notice this. Prettier did whatever it did, I'll try not to change in the future.
